### ISSUE TITLE
Overlapping Markers

### ISSUE THREAD

**Issue by buttersnipps** [humphd/bridge-troll#1]
![image](https://user-images.githubusercontent.com/16315570/37302142-11adb6b0-2601-11e8-8333-e59d45dfe543.png)
Multiple markers get placed on top of each other.

Best solution would be to use [Marker Clustering](https://developers.google.com/maps/documentation/javascript/marker-clustering)

**Comment by humphd** [comment]
Good point.  I've been reworking the UI a bit today, and `master` looks like this at the moment:

<img width="649" alt="screen shot 2018-03-13 at 9 28 02 pm" src="https://user-images.githubusercontent.com/427398/37378436-825ac074-2705-11e8-84a2-8481cce32baa.png">

A few thoughts:

* Eventually I want to lock down the map so you can't move/zoom/etc, and it only works if you actually `move` (walk, car, bus, whatever).  As such, we could set the zoom level to something that will mean that bridges never cluster.
* If we want to try clustering in Leaflet, which is what I've been using, there are plugins like https://github.com/Leaflet/Leaflet.markercluster, https://github.com/SINTEF-9012/PruneCluster, etc. and various tutorials like https://asmaloney.com/2015/06/code/clustering-markers-on-leaflet-maps/

Someone do a PR!

**Comment by humphd** [comment]
Filed #3 to build on this with a mini-map view

**Comment by buttersnipps** [comment]
We can load marker based on the users current location and limit unnecessary api calls to Leaflet.
Locking down the markers based on users location by setting a fix radius will be beneficial don't you think?

Taking into account the bigger picture , we can use a "discover" or "unlock markers"  feature only if the user visited the location. eg . I cant sit at york and see the trolls in downtown Toronto .

I can start working on a something like this if you feel this would be a good idea.
